event/event.go

```diff
@@ -0,0 +1,214 @@
+// Copyright 2019 The Ebiten Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package event is a package that models events that occur during
+// the execution of a program.
+package event
+
+// Event is an interface that custom events should implement.
+// It is empty for now because there are no general methods
+// required of events yet.
+type Event interface {
+}
+
+// KeyCharacter is an event that occurs when a character is actually typed on
+// the keyboard. This may be provided by an input method.
+type KeyCharacter struct {
+	// Code of the key typed.
```

Comment:
Let's make this sentence (Code is/represents ...)

Reply:
Ok